jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/ChecklistTests.java

```diff
@@ -7,31 +7,24 @@
 import static io.github.com.StaticSite.html5Page;
 import static io.github.com.pages.HtmlElementsPage.weather;
 import static io.github.com.pages.HtmlElementsPage.weatherNoLocator;
-import static io.github.epam.html.tests.elements.complex.enums.Weather.Cold;
-import static io.github.epam.html.tests.elements.complex.enums.Weather.Hot;
-import static io.github.epam.html.tests.elements.complex.enums.Weather.Rainy;
-import static io.github.epam.html.tests.elements.complex.enums.Weather.Sunny;
+import static io.github.epam.html.tests.elements.complex.enums.Weather.*;
 import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
 import static java.util.Arrays.asList;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.containsInAnyOrder;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.hasItem;
-import static org.hamcrest.Matchers.hasItems;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.*;
```

Comment:
We shouldn't use wildcard imports. Please, fix it here and in other affected files

Reply:
done:


import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.containsInAnyOrder;
import static org.hamcrest.Matchers.containsString;
import static org.hamcrest.Matchers.hasItem;
import static org.hamcrest.Matchers.hasItems;
import static org.hamcrest.Matchers.hasSize;
import static org.hamcrest.Matchers.not;
import static org.testng.Assert.assertEquals;
import static org.testng.Assert.fail;